src/test/java/com/wikia/webdriver/testcases/communityheadertests/CommunityHeaderTests.java

```diff
@@ -45,36 +57,69 @@ public void testAnonWikiButtons() {
     Assertion.assertTrue(modal.isCreateNewArticleModalVisible());
   }
 
+  @DontRun(language = "szl")
   @Test(groups = {"CommunityHeaderTests"})
   @Execute(asUser = User.USER)
   public void testLoggedInWikiButtons() {
     CommunityHeader communityHeader = new CommunityHeader();
 
     communityHeader.clickWikiActivity();
-    Assertion.assertTrue(driver.getCurrentUrl().contains("Special:WikiActivity"));
+    Assertion.assertStringContains(driver.getCurrentUrl(), "Special:WikiActivity");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Special:WikiActivity" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)